### ISSUE TITLE
[New Feature] Add microphone support

### ISSUE THREAD

**Issue by Silent-Hunter** [mkckr0/audio-share#27]
**Describe the new feature you'd like**
It would be lovely to have bidirectional audio.

**Describe the reason why you want to add the feature**
I want this feature because then I could use my analog wired gaming headset as a wireless one using this app.

**Describe how to implement the feature if you can**
I'm not sure, but I have a feeling Pipewire can do this.

**Comment by mkckr0** [comment]
Your headset plug into the Android phone and you want to let the sound of microphone share to PC?

**Comment by Silent-Hunter** [comment]
Yes, at the same time as audio from PC goes to phone. So I can use it for voice chat while gaming. I think that would be a nice feature!

**Comment by clicky6** [comment]
> **Describe the new feature you'd like** It would be lovely to have bidirectional audio.
> 
> **Describe the reason why you want to add the feature** I want this feature because then I could use my analog wired gaming headset as a wireless one using this app.
> 
> **Describe how to implement the feature if you can** I'm not sure, but I have a feeling Pipewire can do this.

Will not be particularly useful for everyone.

**Comment by nistvan86** [comment]
This would be a really nice feature, as I could use my phone as a wireless proxy for my Bluetooth headset which I can take away with me inside the house while doing conference calls. My WiFi coverage is much better than what my computer's Bluetooth allows.

**Comment by Regela** [comment]
I need this feature too )

**Comment by Elaws** [comment]
Yes, that would make this app as great Audio Relay alternative.
